units/XNB4302/XNB4302_script.lua

```diff
@@ -32,7 +32,7 @@ XNB4302 = Class(NStructureUnit) {
 
                     self:ForkThread(function()
                         self.unit:SetBusy(true)
-                        WaitSeconds(1/self.unit:GetBlueprint().Weapon[1].RateOfFire + .2)
+                        WaitSeconds(1/self.unit.Blueprint.Weapon[1].RateOfFire + .2)
```

Comment:
```suggestion
                        WaitSeconds(1 / self.Blueprint.RateOfFire + .2)
```

There also exists `self:GetWeaponRoF()` but that is used only for structures that can receive adjacency fire rate buffs, which is only `ARTILLERY` category structures.